.github/workflows/devcontainer.yaml

```diff
@@ -16,6 +16,7 @@ concurrency:
 jobs:
   build:
     runs-on: ubuntu-latest
+    timeout-minutes: 14400
```

Comment:
My understanding is that setting a value beyond 6 hours does not make sense for the GitHub hosted runner.